client/view/base/QueryLinks.css

```diff
@@ -0,0 +1,13 @@
+a[data-query] {
```

Comment:
How about styling everything by `.class`  selector (0,1,0) without specifying `tag` selector (0,1,1) to avoid different specificity of selectors?

Reply:
Same argument.
https://github.com/browserslist/browserl.ist/pull/306#discussion_r934416205